spec/models/image_spec.rb

```diff
@@ -1,57 +1,57 @@
 require "spec_helper"
 
 xdescribe Spree::Image, type: :model do
-  let!(:product) { create(:product, name: 'spree_product') }
+  let!(:product) { create(:product, name: "spree_product") }
   let!(:variant) { create(:variant, product: product) }
   let(:spree_image) { Spree::Image.new(viewable_id: variant.id) }
-  let(:image_file) { File.open(Spree::Core::Engine.root + 'spec/fixtures' + 'thinking-cat.jpg') }
-  let(:text_file) { File.open(Spree::Core::Engine.root + 'spec/fixtures' + 'text-file.txt') }
+  let(:image_file) { File.open(Spree::Core::Engine.root + "spec/fixtures" + "thinking-cat.jpg") }
```

Comment:
Using the `join` method is safer than using string concatenation (`Spree::Core::Engine.root.join('spec', 'fixtures', 'thinking-cat.jpg')`. Let's consider using it.